README.md

```diff
@@ -1,10 +1,75 @@
 # indexer
 
-Blockchain indexer and database
+This project indexes data from the regen ledger and stores it in a postgresql database.
+It is built as single python script that spawns independent processes for indexing application specific data.
+This database is currently being used for several purposes in relation to the Regen Marketplace and the Regen Groups UI.
+
+## GraphQL API
+
+The regen-server provides a GraphQL API for accessing the indexer database:
+
+https://api.regen.network/indexer/v1/graphiql
+
+Both the groups-ui and the regen-marketplace use this API to access the data.
+
+## Dependencies
+
+In order to run this project locally you will need the following:
+
+1. a local testnet version of regen-ledger with REST API and RPC enabled
+2. a postgresql database running
```

Comment:
don't we have docker set up for the db? it would be worth adding some more about that here in 2.
or is it just meant for tests?

Reply:
it is worth mentioning that we at least have it as an option for running the database.
i didn't mention it because i haven't been using docker for this project with local dev.
i find it easier to run a postgresql database on my machine outside of docker.

for example, i get a configuration file that i can edit, and i get logs on my disk, both of which end up in regular file-system paths.
both of those are essential when trying to understand a difficult problem with index performance (like we have had here).
and in that scenario, i don't want to have to keep rebuilding a docker image.
i should only have to edit the config file and restart the postgres database.

tangentially:

for similar reasons i like to run the local testnet outside of docker.
overall, i find it easier to grok a system like this when i have more control of how it's running.
OTOH, this is not trivial to get up and running, so the docker setup is beneficial for that.
it does also work nicely for tests.
i personally think that the difficulty in getting this system running provides good learning opportunities that make it easier in the longer term to develop and understand this project.

furthermore, when developing locally it's nice to be able to utilize debuggers.
particularly in python, this is quite a nice thing.
you can set breakpoints from your editor, and can an interactive terminal session.
with other projects i've worked on, getting docker to work with debuggers is non-trivial but possible.

Reply:
i added a section about local dev with docker:
36713397c9c54abb63c28409afe288346bed5815

Reply:
yeah understood, I was just thinking about that because in the migrations/README.md, the first command is docker related: https://www.notion.so/regennetwork/ec16c9e6890f4241990e6575ff9b5cdc?v=bdd8303495c14680a6c92c220ee13458&p=b05daec951a24d52a209d58c7bb06fc3&pm=s
So maybe we should be more explicit here that both locally and docker based pg can work, eg:
```suggestion
2. a postgresql database running, either locally or through docker [ TODO: add a link to the section you added in https://github.com/regen-network/indexer/commit/36713397c9c54abb63c28409afe288346bed5815 ]
```